src/Mortgage/Mortgage.router.js

```diff
@@ -53,14 +53,14 @@ export class MortgageRouter {
   async getMortgagesByBorrower(req) {
     const borrower = server.extractFromReq(req, 'address')
     const status = server.extractFromReq(req, 'status')
-    return await Mortgage.findByBorrower(borrower, status)
+    return Mortgage.findByBorrower(borrower, status)
```

Comment:
As we talked earlier, the expression `return await` is redundant since we are inside an async function unless we are inside a try/catch block and we need to resolve the promise before to know if it has any throwers